src/ast.cpp

```diff
@@ -32,7 +33,68 @@ void CreateTableCommand::execute(Database& database)
 
 void InsertIntoCommand::execute(Database& database)
 {
-  std::cout << *this << std::endl;
+  using namespace white::davisbase::sdl;
+  using common::NullValue;
+  using std::holds_alternative;
+
+  auto table_opt = database.getTable(table_name);
+  if (table_opt == std::nullopt) {
+    throw std::runtime_error("Table doesn't exist");
+  }
+
+
+  std::map<std::string, common::ColumnDefinition> columns_map;
+  std::vector<common::ColumnDefinition> column_defs =
+    table_opt.columnDefinitions();
+
+  for (int i = 0; i < column_defs.size(); i++) {
+    columns_map[column_defs[i].name] = column_defs[i];
+  }
+
+  if (column_names.size() > 0) {
+    for (int i = 0; i < column_names.size(); i++) {
+
+      if (columns_map[column_names[i]] != std::nullopt) {
+
+        auto colValVariant =
+          createColumnValue(columns_map[column_names[i]], values[i]);
```

Comment:
```
/home/travis/build/ozars/white-davisbase/src/ast.cpp:60:68: error: cannot convert ‘std::map<std::__cxx11::basic_string<char>, white::davisbase::common::ColumnDefinition>::mapped_type’ {aka ‘white::davisbase::common::ColumnDefinition’} to ‘white::davisbase::common::ColumnType’
```

`createColumnValue` takes `ColumnType` as its first argument instead of `ColumnDefinition`. `ColumnDefinition` should have a member value for its type (named "type" IIRC).

https://github.com/ozars/white-davisbase/blob/ebc74d34a4e4f0e4ec13c96523298c85e3b3d33a/src/sdl/column_value.hpp#L127-L128